bson/src/main/org/bson/json/Base64.java

```diff
@@ -8,27 +8,26 @@
  *
  *   http://www.apache.org/licenses/LICENSE-2.0
  *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
  */
 
-package com.mongodb.connection;
-
-import com.mongodb.MongoException;
-
-import java.io.UnsupportedEncodingException;
+package org.bson.json;
 
 /**
  * <p>Provides Base64 encoding and decoding.</p>
  * <p>This class implements Base64 encoding</p>
  * <p>Thanks to Apache Commons project. This class refactored from org.apache.commons.codec.binary</p>
  * <p>Original Thanks to <a href="http://svn.apache.org/repos/asf/webservices/commons/trunk/modules/util/">commons</a> project in
  * ws.apache.org for this code. </p>
+ *
+ * @since 3.5
  */
-class Base64Codec {
+public final class Base64 {
```

Comment:
Sounds good.  When we modularize for Java 9 we can mark that package as public only to our own upstream modules.